src/cc2olx/olx.py

```diff
@@ -250,6 +252,18 @@ def process_external_tools_link(item, html):
             html = html.replace(item, external_tool_url)
             return html
 
+        def process_extra_static_files(item, html):
```

Comment:
Please add a docstring here explaining what this function is for.

Reply:
Done.